### PR TITLE
migrated sig-storage-gce presubmits to containerd

### LINKED ISSUE #24851: sig-storage-gce presubmits should migrate to containerd
sig-storage-gce presubmits should migrate to containerd after dockershim removal.

/sig storage

**What should be cleaned up or changed**:
Look for `--env=KUBE_CONTAINER_RUNTIME=docker` in the config file.

**Provide any links for context**:
https://github.com/kubernetes/test-infra/blob/882c0569d9f7dce4ba6bf58618e564649733319f/config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

### PR DESCRIPTION
**Pull request by NikhilSharmaWe**
Fixes #24851

### CODE REVIEW

File: config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml [hunk 1]
```diff
@@ -181,7 +181,7 @@ presubmits:
         - --gcp-node-image=ubuntu
         - --image-family=ubuntu-2004-lts
         - --image-project=ubuntu-os-cloud
-        - --env=KUBE_CONTAINER_RUNTIME=docker
+        - --env=KUBE_CONTAINER_RUNTIME=containerd
```
Comment (endocrimes) [inline config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml:184]:
```suggestion
        - --env=KUBE_CONTAINER_RUNTIME=containerd
        - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.5.9                                                                                                               
        - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.0.3
```

File: config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml [hunk 2]
```diff
@@ -226,7 +226,7 @@ presubmits:
...
```
Comment (endocrimes) [inline config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml:229]:
```suggestion
        - --env=KUBE_CONTAINER_RUNTIME=containerd
        - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.5.9                                                                                                               
        - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.0.3
```

File: config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml [hunk 3]
```diff
@@ -298,7 +298,7 @@ periodics:
       - --gcp-node-image=ubuntu
       - --image-family=ubuntu-2004-lts
       - --image-project=ubuntu-os-cloud
-      - --env=KUBE_CONTAINER_RUNTIME=docker
+      - --env=KUBE_CONTAINER_RUNTIME=containerd
```
Comment (endocrimes) [inline config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml:301]:
```suggestion
      - --env=KUBE_CONTAINER_RUNTIME=containerd
      - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.5.9                                                                                                               
      - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.0.3
```

File: config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml [hunk 4]
```diff
@@ -324,7 +324,7 @@ periodics:
...
```
Comment (endocrimes) [inline config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml:327]:
```suggestion
      - --env=KUBE_CONTAINER_RUNTIME=containerd
      - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.5.9                                                                                                               
      - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.0.3
```

**Review by endocrimes — COMMENTED** [review]
if you specify containerd/runc versions they'll get bumped for you by sig-node folks fairly quickly after releases.

File: config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml (hunk 1 above)
Comment (NikhilSharmaWe) [reply config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml:184]: @endocrimes Thanks for the help. I had one question - 
- If by specifying the versions of the container runtime helps in bumping their versions quickly after releases, then why docker's version was not specified before.

Comment (endocrimes) [reply config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml:184]: @NikhilSharmaWe With docker the version just came from the node image, with containerd we started testing versions more explicitly.

**Review by endocrimes — COMMENTED** [review]
/lgtm

(from a node/containerd config pov)

**Pull request merged**